index.js

```diff
@@ -121,7 +121,10 @@ class DockerExecutor extends Executor {
      * @return {Promise}
      */
     _start(config) {
-        const [buildImage, buildTag] = config.container.split(':');
+        const containerName = config.container.split(':');
+
+        const buildTag = (containerName.length === 1) ? 'latest' : containerName.pop();
+        const buildImage = containerName.join(':');
```

Comment:
Not in this case. `buildImage` is the docker image without the tag suffix. So for the image `node:6`, `buildImage` would be defined as `node`. 

After reading your comment, I noticed something weird. If a user specifies the container from a specific docker registry `my-private.docker-registry.com:1234/repo/image`, the current implementation fails to default the tag to `latest`. Do you know of a better way to detect if the container specifies a particular registry?